master/package.py

```diff
@@ -163,6 +163,15 @@
         env=julia_package_env,
     ),
 
+    # Convert tarball to zip on Windows
+    steps.ShellCommand(
+        name="make .zip",
+        command=["python", "-m", "../commands/tar2zip.py", util.Interpolate("%(prop:local_tarball_name)s")],
```

Comment:
it seems wayward to call a python script within a python script to do this, but looking at the buildbot docs it sounds like `command` is meant to launch shell commands so this is what I did.  I really don't mind as long as it works, but just thought I'd mention.